util/__init__.py

```diff
@@ -1091,3 +1092,19 @@ def process_data(cls, data, bigrams):
 english_bigrams = Bigrams(Counter())
 english_bigrams.proportional = Counter(english_bigram_frequencies)
 
+
+class MoneyUtility(object):
```

Comment:
A simple class that will use whatever heuristics we end up coming up with to turn a string into a Money object.